src/readqps.jl

```diff
@@ -816,5 +835,32 @@ function readqps(qps::IO; mpsformat::Symbol=:free)
 
     endata_read || @error("reached end of file before ENDATA section")
 
+    # Finalize variable bounds
+    # All marked integer variables with no explicit bounds
+    # are converted to Integer variables with bounds [0, 1]
```

Comment:
I think this is solver-dependent. Options being `(-Inf, Inf)`, `[0, Inf)`, or `[0, 1]`. Seems like it should probably be `[0, Inf)`, but we should throw a warning. This is messy.